### PR TITLE
Fixes #15506: Change rudder agent bash autocomplete file to add rudder-pkg completion

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/15506

### CODE REVIEW

**Comment by Fdall** [comment]
PR updated with a new commit

File: etc/bash_completion.d/rudder.sh [hunk 1]
```diff
@@ -1,8 +1,26 @@
+_global() {
+  if [ $COMP_CWORD -ge 1 ]; then
+    case ${COMP_WORDS[1]} in
+      package)
+        PKG_FILE="rudder-pkg.sh"
```
Comment (peckpeck) [inline etc/bash_completion.d/rudder.sh]: I think you need an absolute path here since you don't know the current directory

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
OK, squash merging this PR

**Comment by Fdall** [comment]
OK, squash merging this PR

**Comment by Fdall** [comment]
PR rebased

**Comment by Fdall** [comment]
PR rebased

**Comment by Fdall** [comment]
PR rebased

**Comment by Fdall** [comment]
OK, merging this PR

**Pull request merged**